src/annotations.py

```diff
@@ -0,0 +1,252 @@
+import warnings
+import numpy as np
+import oyaml as yaml
+import pandas as pd
+
+
+def _disambiguate_sonyc_team_annotations(series):
```

Comment:
can yo double check this? i'm getting some weird output with mostly 1s for the gt...